localstorage/firebase-worker.js

```diff
@@ -46,12 +56,21 @@ async function initializeApp(withPersistence) {
   if (withPersistence) {
     console.log('Starting Firestore with persistence.');
     await self.db.enablePersistence({experimentalForce: true}).then(() => {
-      postMessage('takePersistence');
+      postMessage('persistenceTaken');
     });
```

Comment:
Rather than await this, we should perhaps just add a .catch() handler that logs to console.error